test/org/sagebionetworks/bridge/validators/ScheduleValidatorTest.java

```diff
@@ -120,6 +121,25 @@ public void expiresTooShort() {
     }
     
     @Test
+    public void rejectsRepeatingScheduleWithNoExpiration() {
```

Comment:
If you set a cron trigger we always treat it as a recurring schedule. To create a one-time schedule you have to set it to once and use an appropriate delay. No one has asked for something like "one time on Tuesday morning, at 8am, after the user registers..." we can change to support if it comes up.